apps/fz_http/lib/fz_http/gateways.ex

```diff
@@ -23,6 +47,24 @@ defmodule FzHttp.Gateways do
     |> Repo.delete()
   end
 
+  def get_gateway!, do: get_gateway!(name: @default_name)
   def get_gateway!(id: id), do: Repo.get!(Gateway, id)
   def get_gateway!(name: name), do: Repo.get_by!(Gateway, name: name)
+
+  def gateway_config(gateway) do
+    interface = %{
+      address: ["#{gateway.ipv4_address}/32", "#{gateway.ipv6_address}/128"],
+      mtu: gateway.mtu
+    }
+
+    %{
+      default_action: @default_action,
+      interface: interface,
+      peers: Devices.as_settings() |> MapSet.to_list()
+    }
+  end
+
+  def list_gateways, do: Repo.all(Gateway)
+
+  def count_gateways, do: Repo.aggregate(Gateway, :count, :id)
```

Comment:
I think in other contexts this function is called just `count`, so we can keep it the same just to honor the convention:
```suggestion
  def count, do: Repo.aggregate(Gateway, :count, :id)
```